tests/integration/test_mysql_protocol/prepared_statements_test.sql

```diff
@@ -0,0 +1,127 @@
+CREATE TABLE ps_simple_data_types
+(
+    i8    Int8,
+    i16   Int16,
+    i32   Int32,
+    i64   Int64,
+    i128  Int128,
+    i256  Int256,
+    ui8   UInt8,
+    ui16  UInt16,
+    ui32  UInt32,
+    ui64  UInt64,
+    ui128 UInt128,
+    ui256 UInt256,
+    f32   Float32,
+    f64   Float64,
+    b     Boolean
+) ENGINE MergeTree ORDER BY i8;
+
+INSERT INTO ps_simple_data_types
+VALUES (127, 32767, 2147483647, 9223372036854775807, 170141183460469231731687303715884105727,
+        57896044618658097711785492504343953926634992332820282019728792003956564819967,
+        255, 65535, 4294967295, 18446744073709551615, 340282366920938463463374607431768211455,
+        115792089237316195423570985008687907853269984665640564039457584007913129639935,
+        1.234, 3.35245141223232, FALSE),
+       (-128, -32768, -2147483648, -9223372036854775808, -170141183460469231731687303715884105728,
+        -57896044618658097711785492504343953926634992332820282019728792003956564819968,
+        120, 1234, 51234, 421342, 15324355, 41345135123432,
+        -0.7968956, -0.113259, TRUE);
+
+CREATE TABLE ps_string_types
+(
+    s String,
+    sn Nullable(String),
+    lc LowCardinality(String),
+    nlc LowCardinality(Nullable(String))
+) ENGINE MergeTree ORDER BY s;
+
+INSERT INTO ps_string_types
+VALUES ('foo', 'bar', 'qaz', 'qux'),
+       ('42', NULL, 'test', NULL);
+
+CREATE TABLE ps_decimal_types
+(
+    d32         Decimal(9, 2),
```

Comment:
You probably want to use aliases of Decimal (e.g. Decimal32) instead of `Decimal(9,2)` for better readability. https://clickhouse.com/docs/en/sql-reference/data-types/decimal